lib/util/constants.ts

```diff
@@ -6,3 +6,4 @@ export const packageJsonInfo = {
   version: "0.1.0"
 };
 export const messageDispositionTimeout = 20000;
+export const defaultLastUpdatedTime = 259200000; // 3 * 24 * 3600 * 1000
```

Comment:
By its name, its not clear what this constant is going to be used for...

Reply:
It is used to provide the latUpdatedTime parameter while enumerating sessions. It is a required parameter. However, both Anna and me thought that it is better to set some default value for that parameter to provide a better developer experience and make this an optional parameter.

Would `defaultLastUpdatedTimeForListingSessions` be a better name?

Reply:
sounds good. Shall do that.